src/test/java/nextstep/favorite/acceptance/FavoriteAcceptanceTest.java

```diff
@@ -1,8 +1,175 @@
+import nextstep.favorite.application.dto.FavoriteResponse;
+import nextstep.fixture.MemberFixture;
+import nextstep.subway.domain.request.LineRequest;
 import nextstep.utils.AcceptanceTest;
+import nextstep.utils.DatabaseCleanup;
+import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+
+import java.util.List;
+
+import static nextstep.exception.ExceptionMessage.AUTHENTICATION_FAILED;
+import static nextstep.exception.ExceptionMessage.NOT_CONNECTED_EXCEPTION;
+import static nextstep.favorite.acceptance.FavoriteSteps.*;
+import static nextstep.subway.utils.LineTestUtil.지하철_노선_생성;
+import static nextstep.subway.utils.SectionTestUtil.지하철_구간_추가;
+import static nextstep.subway.utils.StationTestUtil.지하철역_생성;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.junit.jupiter.api.Assertions.assertAll;
 
 @DisplayName("즐겨찾기 관련 기능")
 public class FavoriteAcceptanceTest extends AcceptanceTest {
+
+    @Autowired
+    private DatabaseCleanup databaseCleanup;
+
+    long 강남역, 역삼역, 선릉역, 이호선;
+    String accessToken;
+
+    @BeforeEach
+    public void setUp() {
+        databaseCleanup.execute();
+
+        강남역 = 지하철역_생성("강남역").jsonPath().getLong("id");
+        역삼역 = 지하철역_생성("역삼역").jsonPath().getLong("id");
+        선릉역 = 지하철역_생성("선릉역").jsonPath().getLong("id");
+
+        이호선 = 지하철_노선_생성(new LineRequest("2호선", "green", 강남역, 역삼역, 10)).jsonPath().getLong("id");
+        지하철_구간_추가(이호선, 역삼역, 선릉역, 10);
+
+        MemberFixture memberFixture = new MemberFixture();
+        accessToken = memberFixture.getAccessToken();
+    }
+
+    @Test
+    void 즐겨찾기_생성() {
+        // give
+
+        // when
+        ExtractableResponse<Response> response = 즐겨찾기_생성_요청(강남역, 선릉역, accessToken);
+
+        // then
+        FavoriteResponse favoriteResponse = response.as(FavoriteResponse.class);
+
+        assertAll(
+                () -> assertThat(response.statusCode()).isEqualTo(HttpStatus.CREATED.value()),
+                () -> assertThat(favoriteResponse.getSource().getId()).isEqualTo(강남역),
+                () -> assertThat(favoriteResponse.getTarget().getId()).isEqualTo(선릉역)
+        );
+    }
+
+    @Test
+    void 즐겨찾기_조회() {
+        // give
+        즐겨찾기_생성_요청(강남역, 선릉역, accessToken);
+
+        // when
+        ExtractableResponse<Response> response = 즐겨찾기_조회_요청(accessToken);
+
+        List<FavoriteResponse> favorites = response.as(new TypeRef<>() {});
+
+        // then
+        assertAll(
+                () -> assertThat(response.statusCode()).isEqualTo(HttpStatus.OK.value()),
+                () -> assertThat(favorites).hasSize(1),
+                () -> assertThat(favorites.stream().anyMatch(
+                        favorite -> favorite.getSource().getId().equals(강남역)
+                        && favorite.getTarget().getId().equals(선릉역))).isTrue()
```

Comment:
가독성을 위해 assertThat 내부에서 코드를 작성하기 보다 미리 추출해서 검증하면 어떨까요?
```suggestion
                () -> assertThat(출발역).isEqualTo(강남역),
                () -> assertThat(도착역).isEqualTo(선릉역)
```

Reply:
출발역이 강남역이고 도착역이 선릉역인 즐겨찾기가 존재하는지를 확인하기 위한 검증이어서 anyMatch 로 검증하고자 하였는데, 출발역과 도착역은 어떻게 미리 추출할 수 있을까요 ?

Reply:
즐겨 찾기 생성 이후 1개의 즐겨찾기 조회 API 가 있다고 착각했나봅니다 😅 
목록 조회 후 생성한 즐겨찾기를 찾는 코드이니 지금대로 두셔도 좋을 것 같아요